services/QuillLMS/client/app/bundles/Connect/components/activityHealth/activityHealth.tsx

```diff
@@ -0,0 +1,417 @@
+import { matchSorter } from 'match-sorter';
+import 'react-table/react-table.css';
+import request from 'request'
+import _ from 'underscore'
+import stripHtml from "string-strip-html"
+import { CSVLink } from 'react-csv'
+import { connect } from 'react-redux';
+
+import PromptHealth from './promptHealth'
+import NumberFilterInput from './numberFilterInput'
+
+import LoadingSpinner from '../shared/loading_indicator.jsx'
+import { sort, sortByList } from '../../../../modules/sortingMethods.js'
+import { FlagDropdown } from '../../../Shared/index'
+import { filterNumbers } from '../../../../modules/filteringMethods.js'
+import actions from '../../actions/activityHealth'
+
+const CONNECT_TOOL = "connect"
+const ACTIVITY_HEALTHS_URL = `${process.env.DEFAULT_URL}/api/v1/activities/activities_health.json`
+const ALL_FLAGS = "All Flags"
+const NO_DATA_FOUND_MESSAGE = "Activity Health data could not be found. Refresh to try again, or contact the engineering team."
+
+interface ActivityHealthProps {
+  dispatch: Function;
+  activityHealth: {
+    flag: string;
+  }
+}
+
+interface ActivityHealthState {
+  loadingTableData: boolean;
+  flag: string;
+  fetchedData: Array<Object>;
+  promptSearchInput: string;
+  dataToDownload: Array<Object>;
+}
+
+class ActivityHealth extends React.Component<ActivityHealthProps, ActivityHealthState> {
+
+  state = {
+    loadingTableData: true,
+    flag: '',
+    fetchedData: [],
+    promptSearchInput: "",
+    dataToDownload: []
+  };
+
+  componentDidMount() {
+    this.fetchActivityHealthData();
+  }
+
+  shouldComponentUpdate(nextProps, nextState){
+    return !(this.state === nextState)
+  }
+
+  columnDefinitions() {
+    return [
+      {
+        Header: 'Name',
+        accessor: 'name',
+        filterMethod: (filter, rows) =>
+                    matchSorter(rows, filter.value, { keys: ["name"]}),
+        filterAll: true,
+        resizeable: true,
+        minWidth: 200,
+        sortMethod: sort,
+        Cell: cell => (<a href={cell.original.url} rel="noopener noreferrer" target="_blank">{cell.original.name}</a>)
+      },
+      {
+        Header: 'Activity Categories',
+        accessor: 'activity_categories',
+        filterMethod: (filter, rows) =>
+                    matchSorter(rows, filter.value, { keys: ["activity_categories"] }),
+        filterAll: true,
+        resizeable: true,
+        sortMethod: sortByList,
+        Cell: (row) => (
+          <div>
+            {
+              row.original['activity_categories'] ?
+              row.original['activity_categories'].map((ap) => (
+                <div key={ap}>{ap}</div>
+              )) : ''
+            }
+          </div>
+        )
+      },
+      {
+        Header: 'Tool',
+        accessor: 'tool',
+        filterMethod: (filter, row) => {
+          if (filter.value === "all") {
+            return true;
+          }
+          else {
+            return row[filter.id] === filter.value
+          }
+        },
```

Comment:
since you return from the `if` block, you can just do:

```
if (filter.value === "all") { return true }

return row[filter.id] === filter.value
```